libmamba/src/core/channel.cpp

```diff
@@ -231,18 +231,17 @@ namespace mamba
                 // that already contains the platform
                 else
                 {
-                    std::string cleaned_url = "", platform = "";
+                    std::string platform = "";
                     util::split_platform(
-                        KNOWN_PLATFORMS,
+                        get_known_platforms(),
                         value,
                         Context::instance().platform,
-                        cleaned_url,
+                        value,
                         platform
                     );
                     if (!platform.empty())
                     {
                         platforms.push_back(std::move(platform));
-                        value = cleaned_url;
```

Comment:
I changed this because I think we always want the `rstrip()` from `split_platform`.

Reply:
I think all usecases should be consistent, so let's rstrip all the time.